.github/workflows/python-ci.yml

```diff
@@ -40,10 +42,10 @@ jobs:
     - uses: actions/setup-python@v2
       with:
         python-version: ${{ matrix.python }}
-    - working-directory: ./python_legacy
+    - working-directory: ./python
```

Comment:
Can we have the tests run for both python and python_legacy? Maybe just copy this file as `python-legacy-ci.yml` so we can evolve them separately.

Reply:
I agree on evolving them separately in CI and then using ignores to deduplicate.

It's possible we'll still update something in python_legacy for a bit (like a major security concern) or otherwise have a need to run the test flows for it.

Reply:
SG